main.py

```diff
@@ -164,10 +165,20 @@ def is_neutral(status):
     query_result = data["query_result"]
     data = query_result["data"]
     rows = data["rows"]
+
+    # Determine the date range of this dataset.
+    min_timestamp = min([row.pop("min_timestamp") for row in rows])
+    max_timestamp = max([row.pop("max_timestamp") for row in rows])
```

Comment:
nit: No need for the `[` and `]`

I don't see any value in doing the `.pop()` since it doesn't hurt to have these keys later.